client/src/components/App.js

```diff
@@ -15,24 +15,16 @@ import Login from './auth/Login';
 import Header from './layout/Header';
 import Sidebar from './layout/Sidebar';
 import { Container, Col, Row } from 'react-bootstrap';
-import { useDispatch, useSelector } from 'react-redux';
 import './layout/Layout.css';
 
 import TeamPage from './teamPage/TeamPage';
 import PublicationPage from './publications/PublicationPage';
 import { Fragment } from 'react';
-import { fetchUserAction } from '../actions/users';
 const App = () => {
   const urls = {
     dashboard: '/dashboard',
     profile: '/dashboard/profile',
   };
-  const teamId = useSelector((state) => state.team.teamId);
-  const dispatch = useDispatch();
-
-  useEffect(() => {
-    dispatch(fetchUserAction());
```

Comment:
Unfortunately, we need this one. My mistake! I put a confusing action name (it was before our full discussion about remove users totally) Let me revert it back and give an appropriate name ;)

Reply:
Resolved!